.docker/scripts/init-ext.sh

```diff
@@ -13,7 +13,6 @@ pip install -r "$VENV_DIR/src/ckanext-archiver/requirements.txt"
 pip install -r "$VENV_DIR/src/ckanext-dcat/requirements.txt"
 pip install -r "$VENV_DIR/src/ckanext-qa/requirements.txt"
 pip install -r "$VENV_DIR/src/ckanext-qgov/requirements.txt"
-pip install -r "$VENV_DIR/src/ckanext-scheming/requirements.txt"
```

Comment:
so scheming is now required or is it pulled in by harvester?

Reply:
It no longer has a `requirements.txt` file.